### DISCUSSION TITLE
[Question]: Sometimes incomplete event videos

### DISCUSSION THREAD

**Discussion by kankadev** [blakeblackshear/frigate#11471] [Ask A Question]
### What is your question:

Hi,

why do I sometimes have incomplete videos? The first seconds of the video are without motion and the last two seconds are with motion / object but suddenly stopped in middle of the motion.

I have an example here:

https://github.com/blakeblackshear/frigate/assets/84390410/c2237a7b-fa0b-4fc7-8aa3-7ab39b2f6271

The next event video has the same video like above but in longer version, so I have the "same event" two times. One event video is incomplete and the other seems to be okay.

Is this a bug, feature or a misconfiguration?

Thanks a lot.

**Comment by NickM-27** [comment]
> The next event video has the same video like above but in longer version, so I have the "same event" two times. One event video is incomplete and the other seems to be okay.

if they are for the same object type that would be due to the object being lost and re-detected. Perhaps due to miscofiguration
